nemo_curator/utils/script_utils.py

```diff
@@ -497,11 +497,11 @@ def parse_gpu_dedup_args(description: str) -> argparse.ArgumentParser:
         argumentHelper.parser.add_argument(
             "--input-json-id-field",
             type=str,
-            default="adlr_id",
+            required=True,
             help="The name of the field within each json object of the jsonl "
             "file that assigns a unqiue ID to each document. "
             "Can be created by running the script "
-            "'./prospector/add_id.py' which adds the field 'adlr_id' "
+            "'./prospector/add_id.py' which adds the field "
```

Comment:
Nit: can we also update the path to add_id in the comment here to the correct dir.

Reply:
Updated, thanks!